### DISCUSSION TITLE
The Mesh stopped working

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#6470] [Issues]
Hi all,
I wanted to change the resolution of the the remote connection so in the resolution options for All Devices I chose 1920x1080.
The mesh stopped working because of it.
How do I reset this? 
Thanks.

**Comment by si458** [comment]
Are you changing the screen resolution via the display option in windows? If so this is a windows resolution, nothing to do with meshcentral sadly

**Reply by ghost** [reply]
No. I changed the option in the Remote screen where all the devices are being displayed in the Mesh.

**Reply by ghost** [reply]
You have all kinds of options where all the devices are listed such as Quality, Turning on and off the remote for all devices and also resolution and this is what I wanted to change.
I thought it would change the remote resolution and the mesh stopped working out of the blue.

**Reply by si458** [reply]
Oh u mean the desktop view tab where u can see all the desktops, erm no idea? Don't use that view, have u tried changing it bk to canvas view/window view size?

**Reply by ghost** [reply]
I can't do anything the Mesh is not working.
How do you reset the Mesh to Factory Default? There must be a way to do this.
Is there a command I can run on the server to reset to Factory Default?

**Reply by si458** [reply]
What u do u mean the mesh isn't working?
We need screenshots or video of what's happening.
Meshagent running on the remote machines don't just crash and the web ui is the same

**Reply by ghost** [reply]
Suddently it works again. No Idea what happened but yes, it just crashed and now is back again.
But, the resolution option is gone and there is only Agent option so something might have been changed with this resolution option,
I guess the resolution option is dangerous so if it's gone then I guess it's best to leave it as is.
